fabric-rendering-v1/src/client/java/net/fabricmc/fabric/mixin/client/rendering/RenderPipelineSnippetMixin.java

```diff
@@ -30,4 +33,39 @@ class RenderPipelineSnippetMixin implements FabricRenderPipeline.Snippet {
 	public Optional<Boolean> usePipelineDrawModeForGui() {
 		return FabricRenderPipelineInternals.getUsePipelineDrawModeForGui((RenderPipeline.Snippet) (Object) this);
 	}
+
+	@ModifyReturnValue(
+			method = "toString",
```

Comment:
missing `remap = false`?

Reply:
Shouldnt be needed anymore, they are tottaly ingored now: https://github.com/FabricMC/tiny-remapper/commit/0bbcbe4a6ad40eb99c0e6a60f4f21890da4817c2

If someone wants an easy cleanup PR freel free to remove them all.